pages/linux/deluser.md

```diff
@@ -0,0 +1,21 @@
+# deluser
+
+> Delete a user from the system.
+> Note: all commands must be executed as root.
+> More information: <https://manpages.debian.org/deluser.8.en.html>.
```

Comment:
This link is not working for me.

```suggestion
> More information: <https://manpages.debian.org/adduser/deluser.8.html>.
```

Reply:
Strange. In all browsers it redirects to https://manpages.debian.org/buster/adduser/deluser.8.en.html

But adding `adduser/` is a bit more verbose&mdash;for the better, IMO.